weatherAPI/lastupdate.tmp

```diff
@@ -0,0 +1 @@
+{"C:\\Users\\karina.jimenez\\go\\src\\API\\weatherAPI\\controllers":1544228708464370000,"C:\\Users\\karina.jimenez\\go\\src\\WeatherAPI\\weatherAPI\\controllers":1544224075028241800}
```

Comment:
Don't commit files like this one.
Try to avoid to commit files with extension `.tmp  .exe   .zip  .rar  `
Perhaps could be added to the `.gitignore`
```
.idea
/vendor/*
/data
/data_stage
routers/commentsRouter__*
lastupdate.tmp
cmd/integration/integration
*.pprof
*.memprof
.vscode
.DS_Store
cover.out
.zip
.rar
API
```